crates/pipeline-manager/src/runner/error.rs

```diff
@@ -214,21 +215,37 @@ impl Display for RunnerError {
                 )
             }
             Self::AutomatonProvisioningTimeout { timeout } => {
-                write!(
+                writedoc! {
                     f,
-                    "Waiting for provisioning timed out after {}s \
-                    -- this occurs when the required resources requested by the runner \
-                    for deployment took too long to be provisioned",
+                    "
+                    Waiting for provisioning of pipeline resources timed out after {}s. This can be due to two reasons:
+
+                    (1) The provisioning timeout is set too short in comparison to how long it should normally take
```

Comment:
Keep these messages simple and user-focused, not internals focused. For example:

```
Pipeline could not be provisioned within the timeout of {}s. Possible reasons:

1) It takes too long to bring up the pipeline (e.g., cloud resources) compared to the timeout. If so, try increasing `provisioning_timeout_s` in the Pipeline settings.

2) Your cloud environment could not provision some required resources like volumes or containers. The pipeline logs might provide more insight if this is the case.
```